### PR TITLE
Unlock page created (Input component created).

### PR DESCRIPTION
**Pull request by IslomJonMirzakbarov**

### CODE REVIEW

File: apps/web/src/components/Input/Input.module.scss [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    background: #ffffff;
+    width: 100%;
+    height: 100%;
+    padding: 0px 20px;
+    border: 1px solid #e7e7e7;
+
+    // text
+    font-style: normal;
+    font-weight: 400;
+    font-size: 14px;
```
Comment (otabeknos1rov) [inline apps/web/src/components/Input/Input.module.scss:20]: how it comes to use text from template fonts?

File: apps/web/src/components/Input/Input.module.scss [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    line-height: 20px;
+    letter-spacing: 0.02em;
+    color: #888888;
```
Comment (otabeknos1rov) [inline apps/web/src/components/Input/Input.module.scss:23]: use please var colors

File: apps/web/src/components/Input/Input.module.scss [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+    &:focus {
+      outline: 3px solid #dbf2ff;
```
Comment (otabeknos1rov) [inline apps/web/src/components/Input/Input.module.scss:26]: var colors please

File: apps/web/src/components/Input/Input.module.scss [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+      color: #3a3a3a;
+    }
+  }
+
+  .InputValColor {
+    color: #3a3a3a;
+  }
+
+  .ErrorColor {
+    outline: 3px solid #ffdcdc;
```
Comment (otabeknos1rov) [inline apps/web/src/components/Input/Input.module.scss:37]: var colors please

File: apps/web/src/components/Input/Input.module.scss [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+  }
+}
+
+.Error {
+  position: absolute;
+  top: 272.1px;
+
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
```
Comment (otabeknos1rov) [inline apps/web/src/components/Input/Input.module.scss:58]: font from template

File: apps/web/src/components/Input/Input.tsx [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+interface InputProps {
+  error?: boolean;
+  inputVal: string;
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  errorClick?: () => void;
+  placeholder?: string;
+}
+
+const Input = ({error, inputVal, onChange, errorClick, placeholder}: InputProps) => {
+  const [eyeCondition, setEyeCondition] = useState('closed');
```
Comment (otabeknos1rov) [inline apps/web/src/components/Input/Input.tsx:19]: Could we make as const conditions = { CLOSED: 'closed' ,  PASSWORD : 'password' , etc}

File: apps/web/src/components/UploadingPopup/UploadingPopup.module.scss [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+    border-radius: 4px 4px 0px 0px;
+    width: 100%;
+    height: var(--list-item-height);
+    background-color: var(--main-blue);
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 25px;
+    font-weight: 600;
+    font-size: 14px;
```
Comment (otabeknos1rov) [inline apps/web/src/components/UploadingPopup/UploadingPopup.module.scss:25]: use font template please

File: apps/web/src/components/DragAndDrop/index.tsx [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+      },
+    },
+  );
+
+  const onDrop = useCallback((acceptedFiles: any[], rejectedFiles: any[]) => {
+    rejectedFiles?.forEach(file => {
+      file?.errors.forEach((err: any) => {
+        switch (err.code) {
+          case SIZE_ERR.CODE:
+            console.log(err.code);
```
[the remark this answers is not in the document]
Comment (IslomJonMirzakbarov) [reply apps/web/src/components/DragAndDrop/index.tsx:67]: yeah sure

File: apps/web/src/components/Input/Input.module.scss (hunk 1 above)
Comment (IslomJonMirzakbarov) [reply apps/web/src/components/Input/Input.module.scss:20]: just for reference I wrote it, forgot to remove it

File: apps/web/src/components/Input/Input.module.scss (hunk 2 above)
Comment (IslomJonMirzakbarov) [reply apps/web/src/components/Input/Input.module.scss:23]: sorry, created this page in hurry

File: apps/web/src/components/Input/Input.module.scss (hunk 3 above)
Comment (IslomJonMirzakbarov) [reply apps/web/src/components/Input/Input.module.scss:26]: yep, sure

File: apps/web/src/components/Input/Input.module.scss (hunk 4 above)
Comment (IslomJonMirzakbarov) [reply apps/web/src/components/Input/Input.module.scss:37]: there's no var color for this one

File: apps/web/src/components/Input/Input.module.scss (hunk 5 above)
Comment (IslomJonMirzakbarov) [reply apps/web/src/components/Input/Input.module.scss:58]: yeah sure

File: apps/web/src/components/Input/Input.tsx (hunk 6 above)
Comment (IslomJonMirzakbarov) [reply apps/web/src/components/Input/Input.tsx:19]: yeah sure

File: apps/web/src/components/UploadingPopup/UploadingPopup.module.scss (hunk 7 above)
Comment (IslomJonMirzakbarov) [reply apps/web/src/components/UploadingPopup/UploadingPopup.module.scss:25]: if we use font template for 14, it doesn't match font-weight and line-height properties

File: apps/web/src/components/UploadingPopup/index.tsx [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+  return <li className={styles.ListItem}>{file.path}</li>;
+}
+
+function UploadingPopup({ items, setIsPopupShowing }: { items: UploadFile[]; setIsPopupShowing: (value: boolean | ((prevVar: boolean) => boolean)) => void }) {
+  return (
+    <div className={styles.PageContainer}>
+      <div className={styles.TopBar}>
+        <div>Uploading {items.length} Items</div>
+        <button className={styles.CloseBtn} onClick={() => setIsPopupShowing(false)}>
+          X
```
[the remark this answers is not in the document]
Comment (IslomJonMirzakbarov) [reply apps/web/src/components/UploadingPopup/index.tsx:16]: we recently got the design, now we'lll apply it

File: apps/web/src/pages/Lock/Lock.module.scss [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
+        font-weight: 400;
+        font-size: 14px;
+        line-height: 20px;
+        display: flex;
+        align-items: center;
+        color: #888888;
+        margin-bottom: 48px;
+      }
+
+      // .InputPhrase {
```
[the remark this answers is not in the document]
Comment (IslomJonMirzakbarov) [reply apps/web/src/pages/Lock/Lock.module.scss]: yeah sure

File: apps/web/src/components/UploadingPopup/UploadingPopup.module.scss (hunk 7 above)
Comment (otabeknos1rov) [reply apps/web/src/components/UploadingPopup/UploadingPopup.module.scss:24]: got it

File: apps/web/src/components/UploadingPopup/index.tsx (hunk 9 above)
[the remark this answers is not in the document]
Comment (otabeknos1rov) [reply apps/web/src/components/UploadingPopup/index.tsx:16]: great

**Review by dbjowett — APPROVED** [review]

File: apps/web/src/components/Input/Input.module.scss [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+
+  .Input {
+    background: #ffffff;
+    width: 100%;
+    height: 100%;
+    padding: 0px 20px;
+    border: 1px solid #e7e7e7;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 14px;
```
Comment (otabeknos1rov) [inline apps/web/src/components/Input/Input.module.scss:20]: use please template fonts

**Review by otabeknos1rov — APPROVED** [review]

**Pull request merged**